doc/sphinx/source/conf.py

```diff
@@ -34,7 +34,7 @@
 project = 'MOLE'
 copyright = '2023, CSRC SDSU'
 author = 'CSRC SDSU'
-release = '1.0.0'
+release = '1.1.0'
```

Comment:
Hi @pritkc , thank you for taking care of this so promptly. We need to update the RELEASING file to remind ourselves that this `release` has to be updated in the `conf.py`. Thank you